### DISCUSSION TITLE
What's your preferred approach for handling RESTful API versioning?

### DISCUSSION THREAD

**Discussion by VS090** [community/community#67023] [Apps, API and Webhooks] [answered]
### REST API

Question

### Body

URL path, custom headers, or something else? Share your thoughts and experiences below!

**Comment by TrisH0x2A** [comment, marked as answer]
When it comes to maintaining RESTful APIs, versioning is a key consideration. It's essential for ensuring backward compatibility while allowing for the introduction of new features and changes. There are various approaches to versioning, but one common debate centers around how to include the API version in requests.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
